test/test_redisplay.c

```diff
@@ -55,6 +55,22 @@ typedef struct {
 static const PuglRect  redisplayRect   = {2, 4, 8, 16};
 static const uintptr_t postRedisplayId = 42;
 
+static bool
+rectContains(PuglRect outer, PuglRect inner)
+{
+  return outer.x <= inner.x && outer.y <= inner.y &&
```

Comment:
this line is a bit confusing, because the `outer` comes first here, while below it is the `inner`.
I propose to change this to:
```
  return inner.x >= outer.x && inner.y >= outer.y &&
```